mpqp/core/instruction/measurement/pauli_string.py

```diff
@@ -762,4 +1045,6 @@ def __hash__(self):
 Matrix representation:
 `\begin{pmatrix}1&0\\0&-1\end{pmatrix}`
 """
+
+_pauli_atom_dict = {"I": I, "X": X, "Y": Y, "Z": Z}
```

Comment:
Shouldn't we put this at the top of the file (since it is used in the _from... methods)?